programs/cipher/pbcrypt_demo.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+set -e -u
```

Comment:
When #2698 is ready and backported, include `demo_common.sh` and adapt the code here accordingly.